ansible/roles/nethserver/files/nethserver-networks-init

```diff
@@ -1,8 +1,38 @@
-#! /bin/bash
+#!/bin/bash
 
+#Configure link-local addrress
+ifconfig eth0 169.254.$(($RANDOM % 255)).$((1 + $RANDOM % 253)) netmask 255.255.0.0 up
+
+#Fetch the configurations from Droplet Metadata service
+droplet_metadata=$(curl -s http://169.254.169.254/metadata/v1.json)
+
+public_ipv4_address=$(echo $droplet_metadata | jq -r '.interfaces.public[0].ipv4.ip_address')
+public_ipv4_netmask=$(echo $droplet_metadata | jq -r '.interfaces.public[0].ipv4.netmask')
+public_ipv4_gateway=$(echo $droplet_metadata | jq -r '.interfaces.public[0].ipv4.gateway')
+
+private_ipv4_address=$(echo $droplet_metadata | jq -r '.interfaces.private[0].ipv4.ip_address')
+private_ipv4_netmask=$(echo $droplet_metadata | jq -r '.interfaces.private[0].ipv4.netmask')
+
+dns_nameservers=$(echo $droplet_metadata | jq -r '.dns.nameservers | join(",")')
+
+#Reset networks configuration
 > /var/lib/nethserver/db/networks
 
-/etc/e-smith/events/actions/nethserver-base-initialize-db
+/sbin/e-smith/db networks set eth0 ethernet \
+	role green \
+	bootproto none \
+	ipaddr $public_ipv4_address \
+	netmask $public_ipv4_netmask \
+	gateway $public_ipv4_gateway
+
+/sbin/e-smith/db networks set eth1 ethernet \
+	role green \
+	bootproto none \
+	ipaddr $private_ipv4_address \
+	netmask $private_ipv4_netmask
```

Comment:
I'd leave eth1 not enabled: just create the record without role
```suggestion
/sbin/e-smith/db networks set eth1 ethernet \
	role '' \
	bootproto none \
	ipaddr $private_ipv4_address \
	netmask $private_ipv4_netmask
```

The user will be able to enable it from UI, if needed

Reply:
I'm not sure about that, the previous behavior is to have the private interface enabled at boot if present and I think that is what wants a generic DigitalOcean user. :thinking:

Reply:
Fine than, let's keep the old behavior.